src/App.js

```diff
@@ -202,11 +203,12 @@ class App extends Component {
   async getTokenBalance(userAddress,tokenAddress) {
     var tokenInst = new this.state.web3.eth.Contract(erc20ABI,tokenAddress);
     tokenInst.methods.balanceOf(userAddress).call().then(function (bal) {
-        document.getElementById(`balance-${tokenAddress}`).innerHTML = (bal/1000000000000000000).toFixed(7)
+        // document.getElementById(`balance-${tokenAddress}`).innerHTML = (bal/1000000000000000000).toFixed(7)
```

Comment:
why is this still here? lol

Reply:
idk must have missed it. Didn't delete it by the time i pushed